webpack/react_app/components/TargetingHosts/__tests__/TargetingHosts.test.js

```diff
@@ -0,0 +1,40 @@
+import { testComponentSnapshotsWithFixtures } from '@theforeman/test';
+import TargetingHosts from '../TargetingHosts';
+
+const fixtures = {
+  'renders TargetingHosts': {
+    status: '',
+    items: [
+      {
+        name: 'host',
+        link: '/link',
+        status: 'success',
+        actions: [],
```

Comment:
those fixtures could be also move to the `TargetingHosts.fixtures.js` file and reused un other tests,
here they could be consumed as: 

```js
'renders TargetingHosts': {
￼    status: '',
￼    items: items,
```